src/Toml/Codec.hs

```diff
@@ -0,0 +1,57 @@
+{- |
+Copyright: (c) 2018-2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+Reexports functions under @Toml.Codec.*@.
+
+* __"Toml.Bi.Map"__: contains implementation of tagged bidirectional
+  isomorphisms.
+* __"Toml.Bi.Monad"__: core abstraction for bidirectional conversion based on
+  profunctor monads.
+* __"Toml.Codec.Combinators"__: TOML specific combinators based on 'Codec' type
+  from "Toml.Bi.Monad".
+* __"Toml.Codec.Code"__: conversion functions like 'decode' and 'encode'.
```

Comment:
I think this part can be removed. You already provided a nice description of these modules